### ISSUE TITLE
[tailwindcss-react-aria-components] Option 'prefix' does not exist in type

### ISSUE THREAD

**Issue by sefakeles** [adobe/react-spectrum#6514]
### Provide a general summary of the issue here

React Aria styling guide describes [how to prefix all React Aria Components modifiers with a string of choice](https://react-spectrum.adobe.com/react-aria/styling.html#modifier-prefix) as follows.

```javascript
/** @type {import('tailwindcss').Config} */
module.exports = {
  plugins: [
    require('tailwindcss-react-aria-components')({ prefix: 'rac' })
  ],
};
```

However, there is no type definition for `prefix` option.

### 🤔 Expected Behavior?

The [index.d.ts](../blob/main/packages/tailwindcss-react-aria-components/src/index.d.ts) file should have a type definition for `prefix` option.

### 😯 Current Behavior

The following TypeScript error is being received.

`Object literal may only specify known properties, and 'prefix' does not exist in type 'Partial<{ className: string; target: "modern" | "legacy"; }>'.`

### 💁 Possible Solution

Type declaration must be updated as follows.

```diff
-declare function plugin(options?: Partial<{ className: string, target: 'modern' | 'legacy' }>): {
+declare function plugin(options?: Partial<{ prefix: string, className: string, target: 'modern' | 'legacy' }>): {
   handler: () => void
 }
```

### 🖥️ Steps to Reproduce

Use [modifier prefix](https://react-spectrum.adobe.com/react-aria/styling.html#modifier-prefix) in any TypeScript project.

### Version

tailwindcss-react-aria-components 1.1.3

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

### What operating system are you using?

MacOS Sonoma 14.5
